packages/discord.js/src/util/EnumResolvers.js

```diff
@@ -633,6 +634,157 @@ class EnumResolvers extends null {
         return unknownKeyStrategy(key);
     }
   }
+
+  /**
+   * A string that can be resolved to a {@link AuditLogEvent} enum value. Here are the available types:
+   * * GUILD_UPDATE
+   * * CHANNEL_CREATE
+   * * CHANNEL_UPDATE
+   * * CHANNEL_DELETE
+   * * CHANNEL_OVERWRITE_CREATE
+   * * CHANNEL_OVERWRITE_UPDATE
+   * * CHANNEL_OVERWRITE_DELETE
+   * * MEMBER_KICK
+   * * MEMBER_PRUNE
+   * * MEMBER_BAN_ADD
+   * * MEMBER_BAN_REMOVE
+   * * MEMBER_UPDATE
+   * * MEMBER_ROLE_UPDATE
+   * * MEMBER_MOVE
+   * * MEMBER_DISCONNECT
+   * * BOT_ADD
+   * * ROLE_CREATE
+   * * ROLE_UPDATE
+   * * ROLE_DELETE
+   * * INVITE_CREATE
+   * * INVITE_UPDATE
+   * * INVITE_DELETE
+   * * WEBHOOK_CREATE
+   * * WEBHOOK_UPDATE
+   * * WEBHOOK_DELETE
+   * * INTEGRATION_CREATE
+   * * INTEGRATION_UPDATE
+   * * INTEGRATION_DELETE
+   * * STAGE_INSTANCE_CREATE
+   * * STAGE_INSTANCE_UPDATE
+   * * STAGE_INSTANCE_DELETE
+   * * STICKER_CREATE
+   * * STICKER_UPDATE
+   * * STICKER_DELETE
+   * * GUILD_SCHEDULED_EVENT_CREATE
+   * * GUILD_SCHEDULED_EVENT_UPDATE
+   * * GUILD_SCHEDULED_EVENT_DELETE
+   * * THREAD_CREATE
+   * * THREAD_UPDATE
+   * * THREAD_DELETE
+   * @typedef {string} AuditLogEventEnumResolvable
+   */
+
+  /**
+   * Resolves enum key to {@link AuditLogEvent} enum value
+   * @param {AuditLogEventEnumResolvable|AuditLogEvent} key The key to lookup
+   * @returns {AuditLogEvent}
+   */
+  static resolveAuditLogEvent(key) {
+    switch (key) {
+      case 'GUILD_UPDATE':
```

Comment:
Everything else is PascelCase now why this is still UPPER_SNAKE_CASE?

Reply:
Because that's what every other method in the EnumResolvers class does.

Reply:
> Everything else is PascelCase now why this is still UPPER_SNAKE_CASE?

Reply:
I think it's also because discord itself sends all events in UPPER_CASE and it's unnecessary to convert it there.

Reply:
No, Discord sends numbers.

Reply:
> No, Discord sends numbers.

Ah yes, sorry 👍